pyproject.toml

```diff
@@ -91,7 +91,15 @@ linting = [
   "asyncpg-stubs",
   "pylint",
 ]
-test = ["pytest", "pytest-xdist", "pytest-mock", "pytest-cov", "coverage", "pytest-sugar"]
+test = [
+  "pytest",
+  "pytest-xdist",
+  "pytest-mock",
+  "pytest-cov",
+  "coverage",
+  "pytest-sugar",
```

Comment:
we should add this everywhere

Reply:
yep.  I agree.  The output is much cleaner.